activerecord/lib/active_record/explain.rb

```diff
@@ -50,5 +50,25 @@ def render_bind(attr)
 
         [attr&.name, value]
       end
+
+      def build_explain_clause(options = [])
```

Comment:
Minor detail: While a default for `options` makes sense for `explain`, could it be the case that it has no use case in the internal `build_explain_clause`?